client/src/routes/Temple/components/Buttons/SlideButton.tsx

```diff
@@ -1,27 +1,16 @@
 import styled from 'styled-components/native';
 
-import {COLORS} from '../../../../common/constants/colors';
-import {IconType} from '../../../../common/components/Icons';
 import {SPACINGS} from '../../../../common/constants/spacings';
-import IconButton from '../../../../common/components/Buttons/IconButton/IconButton';
-
-type SlideButtonProps = {
-  onPress?: () => void;
-  Icon: IconType;
-  disabled?: boolean;
-  fill?: string;
-};
+import IconButton, {
+  BaseIconButtonProps,
+} from '../../../../common/components/Buttons/IconButton/IconButton';
 
 const SlideButton = styled(IconButton).attrs({
-  fill: COLORS.BLACK,
-})<SlideButtonProps>(({disabled}) => ({
-  backgroundColor: COLORS.GREYLIGHTEST,
-  color: COLORS.BLACK,
+  variant: 'tertiary',
+  elevated: true,
+})<BaseIconButtonProps>(() => ({
   paddingVertical: SPACINGS.EIGHT,
   paddingHorizontal: SPACINGS.TWELVE,
```

Comment:
Maybe don't need this button either, just extend IconButton it in the place we need it and set the padding there